### DISCUSSION TITLE
Labelme to yolo segmentation format

### DISCUSSION THREAD

**Discussion by gyupro** [wkentaro/labelme#1230] [Ideas / Feature requests]
I created a repo to convert labelme format to yolo format

https://github.com/gyupro/LabelMeToYoloSegmentation

I want this feature to be upgraded to the GUI.

Can you do that for users to use GUI easier?

**Comment by wkentaro** [comment]
Like export option from GUI?

**Reply by gyupro** [reply]
yes. yolo nowadays supports segmentation option. The gui supports exporting yolo format would help a lot
